README.md

```diff
@@ -185,11 +185,97 @@ python -m CHAID tests/data/titanic.csv fare sex embarked --max-depth 4 --min-par
+84                                   variable_types=types, **config)
+---> 85   import ipdb; ipdb.set_trace()
+86    
+87        if nspace.classify:
+88            predictions = pd.Series(tree.node_predictions())
+89            predictions.name = 'node_id'
+90            data = pd.concat([data, predictions], axis=1)
+91            print(data.to_csv())
+92        elif nspace.predict:
+```
+
+Running `tree.print_tree()` gives:
+
+``` python
+([], {0: 809, 1: 500}, (sex, p=1.47145310169e-81, score=365.886947811, groups=[['female'], ['male']]), dof=1))
+├── (['female'], {0: 127, 1: 339}, (embarked, p=9.17624191599e-07, score=24.0936494474, groups=[['C', '<missing>'], ['Q', 'S']]), dof=1))
+│   ├── (['C', '<missing>'], {0: 11, 1: 104}, <Invalid Chaid Split>)
+│   └── (['Q', 'S'], {0: 116, 1: 235}, <Invalid Chaid Split>)
+└── (['male'], {0: 682, 1: 161}, (embarked, p=5.017855245e-05, score=16.4413525404, groups=[['C'], ['Q', 'S']]), dof=1))
+    ├── (['C'], {0: 109, 1: 48}, <Invalid Chaid Split>)
+    └── (['Q', 'S'], {0: 573, 1: 113}, <Invalid Chaid Split>)
+```
+
+as show above. The first line is the root node. This means that all the data is present in this node. And the vertical bars originating from a node represents paths to that node's children.
+
+Running `tree.tree_store` will give you a list of all the nodes in the tree:
+
+``` python
+[
+  ([], {0: 809, 1: 500}, (sex, p=1.47145310169e-81, score=365.886947811, groups=[['female'], ['male']]), dof=1)),
+  (['female'], {0: 127, 1: 339}, (embarked, p=9.17624191599e-07, score=24.0936494474, groups=[['C', '<missing>'], ['Q', 'S']]), dof=1)),
+  (['C', '<missing>'], {0: 11, 1: 104}, <Invalid Chaid Split>), (['Q', 'S'], {0: 116, 1: 235}, <Invalid Chaid Split>),
+  (['male'], {0: 682, 1: 161}, (embarked, p=5.017855245e-05, score=16.4413525404, groups=[['C'], ['Q', 'S']]), dof=1)),
+  (['C'], {0: 109, 1: 48}, <Invalid Chaid Split>), (['Q', 'S'], {0: 573, 1: 113}, <Invalid Chaid Split>)
+]
+```
+
+So let's inspect the root node `tree.tree_store[0]`:
+
+``` python
+([], {0: 809, 1: 500}, (sex, p=1.47145310169e-81, score=365.886947811, groups=[['female'], ['male']]), dof=1))
+```
+
+Nodes have certain properties, the have the column that was chosen to split to this node (for a root node there doesn't exist a column, because, well it's the root node and is thus all the data). The second property `{0: 809, 1: 500}` are the members of that node, and represent the frequency of the dependent variable at that node. In this case, it is all the answers in the 'survived' column, as that was the first column past to the program in the command line (`python -m CHAID tests/data/titanic.csv survived`). The next property represents the splitting of the node. What column was chosen to make that split (in this case, `sex`), the p-value of the split and the chi-score and most importantly, which variables in `sex` create the new nodes and the degrees of freedom associated with that split (1, in this case)
```

Comment:
The first sentence here is phrased super awkwardly.  Also  the "what column..." Sentence